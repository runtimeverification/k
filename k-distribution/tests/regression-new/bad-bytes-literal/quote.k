// Copyright (c) Runtime Verification, Inc. All Rights Reserved.
module QUOTE
    imports BYTES
    configuration <k> $PGM:Bool </k>

    rule <k> b""" => b"" </k>
endmodule
