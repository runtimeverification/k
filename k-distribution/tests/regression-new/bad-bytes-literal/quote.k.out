[Error] Inner Parser: Scanner error: unexpected character sequence '"'.
	Source(quote.k)
	Location(6,24,6,25)
	6 |	    rule <k> b""" => b"" </k>
	  .	                       ^
[Error] Compiler: Had 1 parsing errors.
